packages/optimizer/README.md

```diff
@@ -106,11 +163,11 @@ Automatically inject any missing markup required by AMP.
 
 #### `autoExtensionImport`
 
-Automatically import any missing AMP Extensions (e.g. amp-carousel).
+Automatically import any missing AMP Extensions (e.g. amp-carousel). This is not recommended if AMP Optimizer is run in the critical serving path.
```

Comment:
Have you noted all slow options? Would it make sense to be more formal/consistent about how they are documented? A flag of some kind, maybe?

Reply:
The slow options are listed in the table above. Not sure it makes sense to further document these.